env_config/env.docker

```diff
@@ -61,11 +61,13 @@ DB_TZ=America/Vancouver
 # ------------------------------------------------------------------------------
 # KeyCloak URLS
 # ------------------------------------------------------------------------------
-KEYCLOAK_HOST=https://dev.oidc.gov.bc.ca/auth
-KEYCLOAK_REALM=35r1iman
-KEYCLOAK_CLIENT_ID=biohub-platform
-KEYCLOAK_ADMIN_USERNAME=biohubbc-svc
-KEYCLOAK_ADMIN_PASSWORD=
+KEYCLOAK_HOST=https://dev.loginproxy.gov.bc.ca/auth
+KEYCLOAK_REALM=standard
+KEYCLOAK_CLIENT_ID=bio-hub-browser-4230
+KEYCLOAK_INTEGRATION_ID=4230
+KEYCLOAK_ENVIRONMENT=dev
+KEYCLOAK_ADMIN_PASSWORD=LokU5mZqvwgSsnZ4CCNTJspZI3QSC33n
```

Comment:
👀 

There is a button in the keycloak app to regenerate these passwords, so no big deal, but lets remove this of course